src/integrationTest/java/com/hivemq/extensions/influxdb/InfluxDbExtensionIT.java

```diff
@@ -48,18 +48,19 @@ public class InfluxDbExtensionIT {
     private final @NotNull Network network = Network.newNetwork();
 
     @Container
-    private final @NotNull HiveMQContainer hivemq = new HiveMQContainer(OciImages.getImageName("hivemq/hivemq-ce")) //
-            .withExtension(MountableFile.forClasspathResource("hivemq-influxdb-extension"))
-            .waitForExtension("InfluxDB Monitoring Extension")
```

Comment:
We are removing the `waitForExtension`. Are we saying we're implicitly checking that and that it's not necessary to do it?

Reply:
`waitForExtension` is not necessary anymore. This was required in very old HiveMQ versions that started extensions asynchronously (iirc), so that it could happen that HiveMQ already logged "Started HiveMQ ..." before the extensions actually worked. This is not the case anymore in any supported HiveMQ version.